src/Components/Common/UserAutocompleteFormField.tsx

```diff
@@ -80,6 +80,9 @@ export default function UserAutocomplete(props: UserSearchProps) {
         )}
         optionLabel={formatName}
         optionIcon={userOnlineDot}
+        optionImage={(option: UserBareMinimum | UserModel) =>
+          (option as UserModel).read_profile_picture_url || ""
+        }
```

Comment:
if `read_profile_picture_url` is not present in `UserBareMinimum` but backend provides it, then add then move the `read_profile_picture_url` to `UserBareMinimum` instead.

```suggestion
        optionImage={(option: UserBareMinimum) => option.read_profile_picture_url}
```

Reply:
Done..